src/xdocs/config_coding.xml

```diff
@@ -2155,6 +2155,18 @@ try {
         </p>
         <source>
 &lt;module name=&quot;IllegalThrows&quot;/&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+public class Test {
```

Comment:
This and following code: 
This should be indented with 2 spaces to match other examples.

Reply:
Done